src/front-end/src/GlobalState.tsx

```diff
@@ -64,6 +64,7 @@ const modifyAccessToken = (
         },
       },
       user: state.user!,
+      query: state.query!
```

Comment:
Are we sure the global state is the right place for this? Can it be somehow bound to the search page? It really only applies there

Reply:
Ok, I am working on this: 
Perhaps it would be another job for useContext()? Currently I have the SearchBar component within Header/Navigation components which is not ideal for passing down props. And yes, ultimately, the state change  occurs in SearchBar.